### DISCUSSION TITLE
Centering an image

### DISCUSSION THREAD

**Discussion by atsaloli** [py-pdf/fpdf2#727] [Q&A] [answered]
Hello! I am trying to center an image in a pdf.

https://pyfpdf.github.io/fpdf2/fpdf/fpdf.html#fpdf.fpdf.FPDF.image says I can use the `x` argument for this:

>  Align.C can also be passed to center the image horizontally

I tried doing this and got a NameError:

Error:
```
    pdf.image('shirtificate.png', w=pdf.epw, h=pdf.eph,  y=20, x=Align.C)
                                                                 ^^^^^
NameError: name 'Align' is not defined
```

I am learning Python so I might have made a beginner error.  

```
$ pip freeze |grep fpdf
fpdf2==2.6.1
$ python --version
Python 3.11.2
$
```

**Comment by Lucas-C** [comment, marked as answer]
Hi @atsaloli!
You are simply missing an import, no big deal:
```python
from fpdf import Align
```

**Reply by atsaloli** [reply]
Yes, thank you @Lucas-C !

**Reply by Lucas-C** [reply]
I you also got a `NameError: name 'Align' is not defined` as mentioned before, that points to an error due to the use of `Align` without defining what this object is in the script...

Then if you perform a search on `Align` in `fpdf2` documentation, you should find that this enum class is defined there: https://py-pdf.github.io/fpdf2/fpdf/enums.html#fpdf.enums.Align
